pallets/xcm-transactor/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "pallet-xcm-transactor"
 authors = [ "PureStake" ]
 edition = "2021"
-version = "0.1.0"
+version = "1.0.0"
```

Comment:
Major version change, is this the way to increase it?

Reply:
If this version is considered as the first one to be stable, yes, otherwise you have to upgrade to v0.2.0